tests/test_c_parser.py

```diff
@@ -134,20 +137,20 @@ def test_empty_toplevel_decl(self):
 
     def test_coords(self):
         """ Tests the "coordinates" of parsed elements - file
-            name and line numbers, with modification insterted by
-            #line directives.
+            name, line and column numbers, with modification
+            insterted by #line directives.
         """
-        self.assert_coord(self.parse('int a;').ext[0], 1)
+        self.assert_coord(self.parse('int a;').ext[0], 1, 5)
 
         t1 = """
         int a;
         int b;\n\n
         int c;
         """
         f1 = self.parse(t1, filename='test.c')
-        self.assert_coord(f1.ext[0], 2, 'test.c')
-        self.assert_coord(f1.ext[1], 3, 'test.c')
-        self.assert_coord(f1.ext[2], 6, 'test.c')
+        self.assert_coord(f1.ext[0], 2, 13, 'test.c')
```

Comment:
in fact, as the `column` starts at 1, there is no spaces counted in tokens. the 13 starts on the `a`:
```
  TypeDecl: a, []
    IdentifierType: ['int']
```

Again, if you don't agree with the lex column start, don't hesitate to tell me and I will modify the behavior.